vsphere/datadog_checks/vsphere/api.py

```diff
@@ -324,14 +323,44 @@ def query_metrics(self, query_specs):
         )
         return values
 
+    @smart_retry
+    def get_allowed_events(self):
+        if not isinstance(self.config, VSphereConfig) or self.config.include_events is None:
+            exclude_filters = {
```

Comment:
can we set the default value and do this logic in `config.py` instead? This way it only needs to happen once when we initialize the config